### PR TITLE
Burger+factorio

### PR DESCRIPTION
**Pull request by Moltovz**
# What this PR does
- adds the ability to place smelters, stackers, diverters, conveyers, and ore storages

# Why it should be added to the game
- makes mining less fucking awful

# Up next
- make enemies seige drills (syndies and revs?)

### CODE REVIEW

File: code/_core/obj/item/disk/_disk.dm [hunk 1]
```diff
@@ -64,6 +64,61 @@
...
+		"name" = "air_diverter.obj",
+		"effect" = /obj/effect/temp/sparkles,
+		"object" = /obj/structure/interactive/diverter,
+		"cost" = 500
+	)
+
+/obj/item/disk/rcd/orestorage
+	name = "rcd data disk (ore storage)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Air Diverter."
```
Comment (Stalkeros2) [inline code/_core/obj/item/disk/_disk.dm:92]: This isn't air diverter

File: code/_core/obj/item/disk/_disk.dm [hunk 2]
```diff
@@ -64,6 +64,61 @@
...
+		"name" = "ore_storage.obj",
+		"effect" = /obj/effect/temp/sparkles,
+		"object" = /obj/structure/interactive/ore_storage,
+		"cost" = 2000
+	)
+
+/obj/item/disk/rcd/oresmelter
+	name = "rcd data disk (smelter)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Air Diverter."
```
Comment (Stalkeros2) [inline code/_core/obj/item/disk/_disk.dm:103]: This also isn't air diverter

File: code/_core/obj/item/disk/_disk.dm [hunk 3]
```diff
@@ -64,6 +64,61 @@
...
+		"name" = "smelter.obj",
+		"effect" = /obj/effect/temp/sparkles,
+		"object" = /obj/structure/interactive/smelter,
+		"cost" = 2000
+	)
+
+/obj/item/disk/rcd/stacker
+	name = "rcd data disk (stacker)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Air Diverter."
```
Comment (Stalkeros2) [inline code/_core/obj/item/disk/_disk.dm:114]: This also isn't air diverter

**Review by Stalkeros2 — DISMISSED** [review]
Based as FUCK

File: code/_core/obj/item/disk/_disk.dm (hunk 3 above)
Comment (Moltovz) [reply code/_core/obj/item/disk/_disk.dm:114]: Fool you've found out my secret! CTRL+C AND CTRL+V!

**Review by Stalkeros2 — APPROVED** [review]
Nased

**Pull request merged**